src/client/entity-editor/validators/edition.ts

```diff
@@ -133,11 +133,14 @@ export function validateForm(
 	isMerge?:boolean
 ): boolean {
 	let validAuthorCredit;
+	const authorCreditEnable = isIterable(formData) ? formData.getIn(['buttonBar', 'authorCreditEnable'], true) :
+		get(formData, 'buttonBar.authorCreditEnable', true);
 	if (isMerge) {
 		validAuthorCredit = validateAuthorCreditSectionMerge(get(formData, 'authorCredit', {}));
 	}
 	else {
-		validAuthorCredit = validateAuthorCreditSection(get(formData, 'authorCreditEditor', {}));
+		validAuthorCredit = (authorCreditEnable || size(get(formData, 'authorCreditEditor', {})) === 0) &&
+		 validateAuthorCreditSection(get(formData, 'authorCreditEditor', {}), !authorCreditEnable, authorCreditEnable);
```

Comment:
Following my request regarding the double placeholder, could you please also explain why we pass `!authorCreditEnable` and `authorCreditEnable`?

Reply:
It is something like, AC can either be empty or contain null values for author (first row) in case of optional checkbox ticked.
though rather than using the partials we can go with separate function which should handle all these edge cases between client and server validation of AC.

Reply:
I think that might make it easier to follow the code, and perhaps we can out find why I was hitting that validation bug I mentioned.